source/Ship.cpp

```diff
@@ -1205,6 +1242,23 @@ void Ship::Place(Point position, Point velocity, Angle angle, bool isDeparting)
 				bay.ship->SetSwizzle(bay.ship->customSwizzle >= 0 ? bay.ship->customSwizzle : swizzle);
 		}
 	}
+
+	UpdateEscortsState(shared_from_this());
+	isEscortsFullOfFuel = false;
+	isTankerCarrier = false;
+	// Reset tanker carrier state on jump or takeoff which will become set if fighters are deployed.
+	if(HasBays())
+	{
+		if(IsRefueledByRamscoop())
+			isTankerCarrier = true;
```

Comment:
So any ship that is a carrier and has a ramscoop is a tanker carrier?
Seems weird to me, you could have a tanker that has 4000 fuel and only refuels by landing. Idk what the better option is, but this doesnt seem like it.
You don't necessarily want all of your carriers to be giving away fuel if you have that fleet logistic pref on. I'd expect a tanker carrier to have at least 1000 fuel; aka enough fuel for 5 jumps minimum?

Reply:
It’s a pref that’s off by default.  You do want carriers that have ramscoop and fighters with fuel to be capable of distributing fuel.

Reply:
So that means you cant just have one carrier in your fleet that worries about fuelling?

Reply:
The point of fleet logistics is to have a carrier in your fleet that worries about refueling.  It means you don't need extra fuel and ramscoop installed in your battle ships if you don't want to (dedicating more space to armaments, power, or regen).

However, if you do decide to add ramscoop to your fleet it will still work as intended.  They just refuel faster because the fighters complement your fleet with ramscoop.  This enables them to refuel faster.

Reply:
What I meant is we can't have just *one* carrier that does the fuelling, but I guess its fine.
This does get me thinking, what if we're in the heat of battle, will the fighters be used as actual fighters then?